image/index.html

```diff
@@ -0,0 +1,161 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css"
+    />
+    <link rel="stylesheet" href="../GlobalStyles/general.css" />
+    <!-- This is reset.css -->
+    <link rel="stylesheet" href="../GlobalStyles/allstyle.css" />
+    <!-- This style is using the different component styles in other web pages -->
+    <link rel="stylesheet" href="../GlobalStyles/sitestyle.css" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Image</title>
+  </head>
+  <body>
+    <header>
+      <a class="logo" href="../index.html">CUI</a>
+      <span>
+        <span><a href="#">Docs</a></span>
+        <span><a href="#">Examples</a></span>
+      </span>
+    </header>
+
+    <main>
+      <nav>
+        <a href="../GetStarted/index.html">Get Started</a>
+        <hr />
+        <h4>Components</h4>
+        <hr />
+        <ul>
+          <li><a href="../Avatar/index.html">Avatar</a></li>
+          <li><a href="../Alert/index.html">Alert</a></li>
+          <li><a href="../Badge/index.html">Badge</a></li>
+          <li><a href="../Button/index.html">Button</a></li>
+          <li><a href="../card/index.html">Card</a></li>
+          <li><a href="../image/index.html">Image</a></li>
+          <li><a href="../Input/index.html">Input</a></li>
+        </ul>
+      </nav>
+      <section>
+        <h3>Responsive Image</h3>
+        <p>
+          Responsive Image Components are used to dsiplay Images on websites,
+          which are screen responsive. They expand upto the width of the
+          container they are in.
+        </p>
+        <hr />
+        <h4>Round Image</h4>
+        <p>
+          Round Images are the circular, with a default dimension of 150px X
+          150px.
+        </p>
+        <div class="container flex-container">
+          <img src="./Assests/img1.jpg" alt="" class="circle" />
+          <img src="./Assests/img2.jpg" alt="" class="circle" />
+        </div>
+        <div
+          class="container iframe"
+          style="
+            height: 300px;
+            background-color: transparent;
+            padding-left: 0px;
+            padding-right: 0px; ;
+          "
+        >
+          <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28249%2C237%2C212%2C1%29&t=twilight&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=0px&dsblur=4px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=IBM+Plex+Mono&fs=18px&lh=160%25&si=false&es=2x&wm=false&code=%253Cimg%2520src%253D%2522.%252Fimg1.jpg%2522%2520alt%253D%2522%2522%2520class%253D%2522circle%2522%2520%252F%253E"
+            style="
+              width: 680px;
+              height: 103px;
+              border: 0;
+              transform: scale(1);
+              overflow: hidden;
+            "
```

Comment:
You can move the iFrame styling to one of the common stylesheets.

Reply:
Thanks, Tanveer for your valuable feedback .